test/test_suite.bats

```diff
@@ -262,8 +262,20 @@
   [[ ${lines[3]} == "Available arguments:" ]]
 }
 
+@test "No WARNING messages in pihole-FTL.log (besides known capability issues)" {
+  run bash -c 'grep "WARNING:" /var/log/pihole-FTL.log | grep -c -v -E "CAP_NET_ADMIN|CAP_NET_RAW|(Starting pihole-FTL as user root is not recommended)"'
+  printf "%s\n" "${lines[@]}"
```

Comment:
Is this just for debugging?

Reply:
What do you mean? The `printf`will report the result of the `grep -c` command.

Reply:
I meant, is this output only shown and used if there are warnings? It may not be needed if bats will already show stdout of the first command.

Reply:
`bats` doesn't show the output of the first command in its own.

Test code:
```
@test "No WARNING messages in pihole-FTL.log (besides known capability issues)" {
  run bash -c 'grep "WARNING:" /var/log/pihole-FTL.log | grep -c -v -E "CAP_NET_ADMIN|CAP_NET_RAW|(Starting pihole-FTL as user root is not recommended)"'
  printf "%s\n" "${lines[@]}"
  [[ ${lines[0]} == "0" ]]
}

@test "No WARNING messages in pihole-FTL.log (besides known capability issues) - no printf line" {
  run bash -c 'grep "WARNING:" /var/log/pihole-FTL.log | grep -c -v -E "CAP_NET_ADMIN|CAP_NET_RAW|(Starting pihole-FTL as user root is not recommended)"'
  [[ ${lines[0]} == "0" ]]
}
```

Output:
```
[...]
ok 27 Fail on invalid argument
ok 28 Help argument return help text
not ok 29 No WARNING messages in pihole-FTL.log (besides known capability issues)
# (in test file test/test_suite.bats, line 268)
#   `[[ ${lines[0]} == "0" ]]' failed
# 1
not ok 30 No WARNING messages in pihole-FTL.log (besides known capability issues) - no printf line
# (in test file test/test_suite.bats, line 273)
#   `[[ ${lines[0]} == "0" ]]' failed
ok 31 No ERROR messages in pihole-FTL.log
ok 32 No FATAL messages in pihole-FTL.log
[...]
```

Note the missing `1` in the second test output.